Source/Source/CompGuest.cs

```diff
@@ -115,7 +117,7 @@ internal void ClearOwnership()
 
             // Calling this method directly crashes the game (infinite loop, somehow). So here's a copy.
```

Comment:
I don't understand this comment - Couldn't recreate or anything, I also do not understand the need to sort after removing a pawn, assuming the list is already sorted prior to removal, it should remain so after as well.

Reply:
Sorting is there, because it was there in the original method (CompAssignableToPawn.TryUnassignPawn). But it does indeed make no sense.
Why calling the original method doesn't work correctly, is probably because it's overridden.